card/lib/card/cache/persistent.rb

```diff
@@ -96,17 +96,13 @@ def read key
       # @param key [String]
       # @param attribute [String, Symbol]
       def write_attribute key, attribute, value
-        if @store && (object = read key)
+        return value unless @store
+        @store.send(:local_cache).clear # clear rails local cache
```

Comment:
this should help dramatically with view clearing.  seems to be working much closer to expectations.

I didn't even realize the rails caching had a local (middleware) layer.